newsfragments/www-fix-forcefield-choicestring-multiple-selection.bugfix

```diff
@@ -0,0 +1 @@
+Fix ``ChoiceStringParameter`` fields with ``multiple`` to match standard selection behaviour (would deselect a selected item on click, now unselect all except this item).
```

Comment:
Wouldn't this break multiple selections? The description could be improved a bit if that's not the case.

Reply:
I made the change in reaction to this comment: https://github.com/buildbot/buildbot/issues/7773#issuecomment-2273690076

Since it referenced w3school, I assumed it was a convention (I'm really not a frontend person, so I have no base knowledge here).

That's interesting that your link does not follow this, however, the widget has checkboxes and highlighting which provide correct feedback on this behavior to the user.
With the simple select, I think it should follow the basic standards.

Or, I can rework this to use the `react-select` component that you okayed [here](https://github.com/buildbot/buildbot/pull/7984#issuecomment-2377213812).

Reply:
Understood. The example on w3schools is I think less intuitive because the only way to select multiple options is to use Ctrl button. Most likely this does not work on touch-based interfaces.

I think a proper component such as `react-select` is better.

Reply:
react-select is a bit annoying to work with, but doable.
I added a video of usage.

Note that this also make it possible to combine `multiple` and `strict`.
Should I add a newsfragment? This limitation was not clearly documented before, not did it generate a config error, it'd just not display the text input box

Reply:
>  Should I add a newsfragment? This limitation was not clearly documented before, not did it generate a config error, it'd just not display the text input box

Yes, please do. It's still a user visible change even if the previous behavior wasn't documented.